src/main/java/com/autotune/common/k8sObjects/Metric.java

```diff
@@ -33,12 +37,15 @@ public final class Metric {
     private final String name;
     private final String query;
     private final String datasource;
+    @SerializedName("value_type")
     private final String valueType;
+    @SerializedName("kubernetes_object")
     private String kubernetesObject;
-    private final LinkedHashMap<String, EMMetricResult> trialSummaryResult = new LinkedHashMap<String, EMMetricResult>();
+    private final LinkedHashMap<String, EMMetricResult> trialSummaryResult = new LinkedHashMap<>();
     private EMMetricResult emMetricResult;
-    private final LinkedHashMap<String, LinkedHashMap<String, LinkedHashMap<Integer, EMMetricResult>>> cycleDataMap = new LinkedHashMap<String, LinkedHashMap<String, LinkedHashMap<Integer, EMMetricResult>>>();
-    private HashMap<String,AggregationFunctions> aggregationFunctionsMap;
+    private final LinkedHashMap<String, LinkedHashMap<String, LinkedHashMap<Integer, EMMetricResult>>> cycleDataMap = new LinkedHashMap<>();
```

Comment:
@msvinaykumar EM specific information should be moved outside of this class as this is a global class used by both Analyzer and EM. But this is for a separate PR.

Reply:
Have raised issue #553 to keep track of this